spec/features/h2_versioned_object_creation_spec.rb

```diff
@@ -137,5 +137,17 @@
                                                   retry_wait: 10)
 
     expect(page).to have_text('adding file (Public version 2)')
+
+    # Go to public version 1, which can be withdrawn
+    click_link_or_button 'Public version 1'
+    expect(page).to have_text('You are viewing an older version.')
+    accept_confirm 'Once you withdraw this version, the Purl will no longer display it. Are your sure?' do
```

Comment:
Are you sure? 🙂 typo

Reply:
Looks like the typo is in the code: https://github.com/sul-dlss/argo/blob/b13a71371b78c3834e86d12f5c43014313688988/app/components/show/controls_component.rb#L163

Reply:
@justinlittman ☝🏻